test/wasm2asm.asserts.js

```diff
@@ -22,61 +22,34 @@ function asmFunc(global, env, buffer) {
   
  }
  
- function $1(x, y) {
-  x = x | 0;
-  y = y | 0;
-  return x + y | 0 | 0;
+ function $1($0, $1) {
```

Comment:
why did the names change from `x => $0`? the former seems nicer?

Reply:
I think this is probably due to enabling optimizations by default, but if that's backed out this'll probably get fixed